code/modules/mob/living/simple_animal/hostile/human/_humanoid.dm

```diff
@@ -31,6 +31,7 @@
 	var/list/items_to_drop = list()
 
 	var/list/visible_items = list()
+	var/persist = 0
```

Comment:
I don't want my nazi to become a carbon corpse when it dies. humanoid mobs are qdel()'d when they die. It's either adding that which can later be used by more mobs, or doing some awful snowflake shit.